src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -70,6 +70,11 @@ class TwigPreprocessor
      */
     protected $curatorenClient;
 
+    /**
+     * @var array
+     */
+    public $fallbackImages;
```

Comment:
Please make this `private`. There's no need for external classes to have access to this